### DISCUSSION TITLE
What is the folder _PGMA/Cast/Faces for?

### DISCUSSION THREAD

**Discussion by fourstix** [CodyBerenson/PGMA-Modernized#265] [Q&A]
I noticed that there's a _PGMA/Cast/Faces folder.  What is this folder used for?  

Would it be possible for one to put actor head-shot pictures there and have them read by the agent instead of relying on IAFD?

**Comment by JPH71** [comment]
That was the plan...
I am still waiting for the code from someone...
This was for actors who do not have pictures on iafd...
With the 403 issues, I am going to extend this to store pictures scraped...
and thus avoid extra reads on iafd...
But first i need to get the code to enable this...

**Comment by fourstix** [comment]
I think it's a great idea to cache the images there to avoid having to repeatedly scrap the same image from IAFD.  

IAFD has become so bad for me, that I would be happy to *manually* place the actor images in /Cast/Faces if you want to add the code to read cached images from there and document the format.   

For older films, IAFD doesn't have many actor images, so it would be nice to be able to put an actor's image there, even if the actor image isn't found on IAFD.

**Comment by JPH71** [comment]
That was the plan... the guy who was to provide the code was a bit of a
prima Donna... said it was difficult and ge had exams on... and told me I
would have to wait...
I will nother him this week again as exam season is over...
